ccm_web/client/src/pages/BulkSectionCreate.tsx

```diff
@@ -333,6 +333,7 @@ function BulkSectionCreate (props: CCMComponentProps): JSX.Element {
       }
       handleParseSuccess(lines)
     }).catch(e => {
+      console.log(e)
```

Comment:
Do you want to leave this here? if yes, can we make it console.error(e). I think in past that's what we agreed.